domains/aniket-thapa.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "aniket-thapa",
+           "email": "aniketthapa04@gmail.com",
+           "discord": "1249367001961463962"
+        },
+    
+        "record": {
+            "CNAME": "aniket-thapa.portfolio"
```

Comment:
I don't think portfolio is a TLD, please add another record which points to your website.